Makefile

```diff
@@ -811,7 +811,7 @@ $(RUNME) $(BUILD_YML):
 	cp pkg/eve/$(@F) $@
 
 EVE_ARTIFACTS=$(BIOS_IMG) $(EFI_PART) $(CONFIG_IMG) $(PERSIST_IMG) $(INITRD_IMG) $(INSTALLER_IMG) $(ROOTFS_IMG) $(SBOM) $(BSP_IMX_PART) fullname-rootfs $(BOOT_PART)
-eve: $(INSTALLER) $(EVE_ARTIFACTS) current $(RUNME) $(BUILD_YML) | $(BUILD_DIR)
+eve: pkgs $(INSTALLER) $(EVE_ARTIFACTS) current $(RUNME) $(BUILD_YML) | $(BUILD_DIR)
```

Comment:
I sometimes do make pkg/pillar rootfs eve.
With this change that will rebuild all of pkg/* from source, which will take an hour or so more.
Why do you need to add pkgs as a dependency here?

Reply:
Otherwise it fails on the pipeline

But:

1. It should not take an hour
2. `make -j 8 eve` would help a bit
3. I guess the way to fix it, is changing `.github/workflows/build.yml` (but let's wait first until eden tests are finished; will do tomorrow)

Reply:
By "pipeline" you mean our GH workflows? I thought those explictly do make pkgs

Reply:
I removed the commit and now the pipeline fails with: https://github.com/lf-edge/eve/actions/runs/9364843044/job/25779040231?pr=3953

Reply:
> I removed the commit and now the pipeline fails with: https://github.com/lf-edge/eve/actions/runs/9364843044/job/25779040231?pr=3953
> 
> ```
> Unable to find image 'lfedge/eve-mkconf:46f3092ae249ee00da2612e368c5e7637922376d' locally
> ```

Is it only pkg/mkconf which needs to be rebuilt? Or are there others?

Reply:
There are more, the next one is:
```
docker: Error response from daemon: manifest for lfedge/eve-mkrootfs-squash:1c718d751d9f9e57b5e45db7ee07895973c3521f not found: manifest unknown: manifest unknown.
```

Unfortunately I cannot reproduce this problem locally (I thought it is this `PRUNE=1` but it is a different thing).